conn.go

```diff
@@ -62,14 +62,16 @@ func (c *otConn) Ping(ctx context.Context) (err error) {
 	}()
 
 	if c.cfg.SpanOptions.Ping {
-		var span trace.Span
-		ctx, span = createSpan(ctx, c.cfg, method, false, "", nil)
-		defer func() {
-			if err != nil {
-				recordSpanError(span, c.cfg.SpanOptions, err)
-			}
-			span.End()
-		}()
+		if c.cfg.SpanOptions.OmitSpan == nil || !c.cfg.SpanOptions.OmitSpan(ctx, c.cfg, method, false, "", nil) {
```

Comment:
`c.cfg.SpanOptions.OmitSpan == nil || !c.cfg.SpanOptions.OmitSpan(ctx, c.cfg, method, false, "", nil)` shows many times.

Could you please wrap it up with a little util function like `omitSpan` or `spanFilter` to prevent copying codes?

pseudo code
```go
func omitSpan(ctx context, spanOptions Option, method Method, query string, args []driver.NamedValue) bool {
    return spanOptions.OmitSpan == nil || !spanOptions.OmitSpan(ctx, method, query, args)
}
```